src/Basho/Riak/Bucket.php

```diff
@@ -63,7 +67,9 @@ public function getName()
      * Returns the buckets R-value If it is set,
      * otherwise return the R-value for the client.
      *
-     * @return integer
+     * @param null $r
+     *
+     * @return int
      */
     public function getR($r = null)
```

Comment:
Just gutted that crap and replaced it with a ternary operator in delete, store, reload, and getsiblings. :)